src/api/embed_helpers.cc

```diff
@@ -129,12 +130,21 @@ CommonEnvironmentSetup::CommonEnvironmentSetup(
   {
     Locker locker(isolate);
     Isolate::Scope isolate_scope(isolate);
+    HandleScope handle_scope(isolate);
+
+    TryCatch bootstrapCatch(isolate);
+    auto print_Exception = OnScopeLeave([&]() {
+      if (bootstrapCatch.HasCaught()) {
+        PrintCaughtException(
+            isolate, isolate->GetCurrentContext(), bootstrapCatch);
```

Comment:
The embedder code has its own error handling mechanism, it would be a bit nicer to use that here instead of unconditionally printing to stderr:

```suggestion
      node::Utf8Value reason(
          isolate,
          bootstrapCatch.Exception()->ToDetailString(
              isolate->GetCurrentContext())
              .FromMaybe(Local<String>()));
      errors->push_back(reason.ToString());
```

(or alternatively giving `PrintCaughtException()` to print errors to `std::string` instead of stdio streams only)